src/conditionals/simple_programs_with_decision_points/hard/q1/AbsolutelySureTest.java

```diff
@@ -0,0 +1,34 @@
+package conditionals.simple_programs_with_decision_points.hard.q1;
+
+import global.BaseTest;
+import global.variables.Clause;
+import global.variables.clauses.*;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class AbsolutelySureTest extends BaseTest {
+    int input = 5;
+
+    public Clause[] testSentence() {
+        return new Clause[]{
+                new StringLiteral("Enter a number: "),
+                new NewLine(),
+                new IntegerLiteral(input, "returnedValue")
+        };
+    }
+
+    public void runMain() {
+        provideInput(String.valueOf(input));
+        AbsolutelySure.main(new String[]{});
+    }
+
+    @Test
+    public void negativeValueTest() {
+        provideInput("-46");
+        AbsolutelySure.main(new String[]{});
+        String output = getOutput();
+        int contains = output.indexOf("-46");
```

Comment:
you can put -46 in its own variable and then provideInpit(variable) and indexOf(variable) to look less hardcoded